### DISCUSSION TITLE
The Positioning and Vision of RingDAO—Establishing Crypto Connections for communities

### DISCUSSION THREAD

**Discussion by hackfisher** [ringecosystem/discussion#8] [General]
## Background
RingDAO is a decentralized autonomous organization dedicated to developing and investing in Web3.0 applications and tools. The goal is to **establish crypto connections for communities**, transcending the boundaries of blockchains and social groups through technology, products, and services.

However, we have come to realize that focusing solely on specific technologies and products is not enough. The mission of RingDAO must integrate more **altruistic values**—providing solutions not only for our users but also creating tangible value for other communities. To achieve this, we need to rethink our approach, starting from the needs of users and communities rather than purely from a technical standpoint. At the same time, we need to assess and optimize the limited resources we currently have From an investment perspective (economic and social benefits).

---

## What Resources Do We Have?
1. **A Global, Passionate Community**  
   RingDAO is supported by a global network of highly engaged community members eager to contribute.

2. **Connection Technologies and Solutions**  
   We have established technologies and solutions, such as Darwinia, Msgport, Paralink, Helixbox, and Paralink, to help break down the barriers between blockchains and communities.

3. **Ecosystem Products and Service Providers**  
   Tools like Helixbox BAAS, XAccount DAO Tool, and network of service providers have the potential not only to improve user experiences but also to address broader needs.

---

## Challenges We Face
- **Shifting Focus from Technology to Users**  
   Should we continue to seek user scenarios starting from technology? Or should we prioritize understanding the needs of external communities and focus on providing practical tools and services?

- **Organizing and Optimizing Key Services**  
   Before making further investments or developments, we must focus on curating and refining our existing resources to effectively and directly address the needs of external communities. This requires carefully evaluating and identifying the return value of potential investments, particularly ensuring they deliver clear service value. Only by doing so can we attract more external supporters and investors, leveraging our existing resources to their full potential. Inefficient investments, such as aimless spending on external services, engineering development, or similar activities, should be avoided as they consume valuable time and resources without delivering meaningful impact.

- **Innovating Community Services and Formats**  
   Beyond technical products, how can we adopt new community services, such as collaborative channels or cross-community initiatives, to better understand market demands and enhance connections with other communities?

  One of the current challenges is that community collaboration often relies too heavily on complex, non-scalable development integrations and technical research rather than “product-ready” integrations and interfaces. To address this, communities should focus on creating lightweight, scalable service formats. These services do not need to be apps or even technology-related; they could include periodic consultation reports between external communities, organized conferences, or cohort-based learning groups.

  High-value, actionable applications and needs are often discovered during these interactions. Importantly, such lightweight service formats require only basic tools to improve efficiency rather than complex technical development. They primarily rely on organizational and outreach capabilities. This approach can significantly enhance RingDAO’s influence, as these services can be implemented immediately, making it easier for external communities to provide sponsorship and support.

---

## Preliminary Vision: Starting from External Community Needs
We propose that RingDAO should center its efforts on the needs of external communities, offering the following practical services and organizational innovations:

### 1. Example for illustration: Optimizing and Organizing Resources (New Ideas for better organised and guided community)
- **Crypto Community Collaboration Channels**  
  By leveraging tools like [Matterbridge](https://github.com/42wim/matterbridge), we can facilitate technical support and interaction across different external communities, serve them and learn them. Combined with our existing ambassador program, this approach can drive resource integration and community growth.

- **Crypto Community Index Fund**  
  Index Fund(AMM pools) consist of tokens from different community tokens, build for users who are interested in related different communities, strengthening inter-community relationships and liquidities. Inspired by https://app.devcon.org/schedule/8DCP9K
  
- More TBD

### 2. Practical App and Tools of Existing Technologies (Self-owned or Ecosystem Partners)
- **Msgport**: Cross-chain messaging orchestration service for EVM developers
- **Helixbox BAAS**: Simplified deployment of cross-chain bridges for EVM chains.
- **Helixbox Bridge**: Cross-chain swap DApp  
- **DAO Tool(split from pre. XAccount)**: DAO cross-chain proposal assistant tool  
- **Paralink**: Polkadot parachain and XCM asset connection app
- **Darwinia**: EVM platform connecting to Polkadot and EVM chains, hosting RingDAO ecosystem liquidity and apps. 
- More TBD

---

## Discussion Questions
1. **Is the core vision of RingDAO clear? Does it need adjustments?**  
2. **How can we better leverage existing technologies to create tangible value for communities?**  
3. **What additional services or organizational formats could help RingDAO realize its vision?**  
4. **What key areas should we prioritize during this process?**
5. **If we want to make RingDAO sustainable, how can we improve investment returns, and how long should the runway be maintained? What service expenditures are ineffective and should be optimized?**

**Comment by Divblaze** [comment]
The core vision of RingDAO is clear but are there any plans to increase usability of the products listed as I feel products like Helixbox should have more usability as it solves a major issue of crosschain bridge and how about RingDAO set up a medium for new Developers to come build on Darwinia network with support from RingDAO. There are a lot of things that can be put in place to boast RingDAO ecosystem

**Reply by hackfisher** [reply]
Definitely, Helixbox has already realized there’s plenty of room to spice up the UX. In fact, they’re at the stage where they can start baking some ideas to onboard Web2 users. Something exciting is definitely cooking!

**Comment by JuneCribb** [comment]
It's great to see RingDAO shift from tech to users and the key areas to prioritize are the usability of products mentioned and the community. 

A good return on investment happens often when we have a vibrant and active community relying on our products.

We want to bring external communities and collaborate which can happen through AMA, following trends, and more. But while doing this, users don't want to know how the tech was built but what it can do for them. 

We’re solving a good problem but we focus on shilling the tech rather than selling a solution to users. 

We should make an adjustment there. 

💜

**Comment by sonoferin2000** [comment]
I think it is key to have a clear understanding of what is needed befor plowing resources in a particular direction. knowing the needs doesnt cost us anything,but is very valuble starting point. probobly best to start by sending out surveys to a wide variety  of communities and projects  to identify common pain points. that way you get a better view of what actual needs there are,and judge which ones are most common that needs addressing. direct resources toward high-impact areas. as for the assets we already have, I agree whole heartedly with JuneCribb,,we dont need to shill the inner workings of the tech itself, people dont want to be bogged down with all the technical details, people are more interested in the solution it brings.  wrap it up in a pretty package for them.  they want a finished product with the least hassle.

**Comment by SasoLithops** [comment]
Thank you @hackfisher for this debate. I will reply to discussion questions.

**1., 4., 5.**
I think core vision of RINGDAO is not clear. It changes a lot. I feel like we have too many things we are working on. I would much rather we focus on one thing and work on others in background first, before marketing them. 
I love idea of Crypto community index fund. I was part of ION DAO in Cosmos eco and its sole project was Cosmos index fund. Imagine if we have Crypto Index Fund where community votes for tokens that get in or out of index fund. All possible communites could help in voting. KTON could be the voting token as it misses additional utility and RING could be used to mint token...and both Ring and KTON should be in index by default(percentage decided)....Part of Ring for minting token could get burned, part for treasury, part for fund changes governance voters.... Having liquid staked tokens in the index meaning tokens that also earn staking rewards is huge. We could primarily focus soley on that. Part of fees used for minting Index token would go to further development and....sorry so many ideas on this....that all is for another debate if liked by community..... @sonoferin2000 and I have been playing with similar idea in KTON DAO and are surely up for it...

**2.**
I wouldn't force anymore technology details into communities as most don't understand it and even more don't care.  All that is well said in @JuneCribb and @sonoferin2000 post. Those that do care will ask. I have some basic knowledge what we work on, but get lost easily. I can only imagine how community feels that asks or DM us as they are so lost with all the updates that usually offer poor guides so moderators have a lot of work cut out for them. I personally miss us ambassadors, mods and willing community members to help in testing product, updates before it goes live.....other cahins have huge number of community testnet participants we can count them on one hand and us being tech orientated project...This is how communites should understand tech through testing....I can share more of this if desired....

**3.**
I have no knowledge on this. Probably being on many tier 1 or tier 2 conferences to introduce our project.

**Reply by hackfisher** [reply]
Thank you, I agree with you that this vision is too complicated. We need to make some adjustments and remove the part about building applications and tools. Instead, we should simplify rather than add more. At this stage, RingDAO might need to shift toward being an investment-oriented DAO, perhaps still focusing on cross-chain initiatives. It could resemble GnosisDAO in some ways, potentially investing in cross-chain applications and leveraging an increasingly “fat” application layer to gain returns through application tokens. The projects, operations, and governance of these applications should be handled by their own teams, while RingDAO provides ecosystem support and investment. Applications could also issue their own tokens to attract other investors.

**Reply by hackfisher** [reply]
https://github.com/orgs/darwinia-network/discussions/1627

**Reply by SasoLithops** [reply]
Thank you @hackfisher 
Love this: https://github.com/orgs/darwinia-network/discussions/1627
@sonoferin2000 and all interested let's help and work on this together with our builders. Only together we strong.

**Reply by hackfisher** [reply]
How about:

> Cross and Connect Chains: Delivering a lightweight and open-source interoperability stack for Web3 applications.

**Reply by SasoLithops** [reply]
How would that benefit Ring holders and RING DAO? I may not understand well. Can you give a more detailed explanation or example of what Apps could do with it.
I think helping easy connect web2 to web3 is good direction to start.

**Comment by SasoLithops** [comment]
Not sure about us turning into VCs(if I understood correctlly investing in cross chain applications). I am not familiar with Gnosis DAO, so I can not comment on that. 
If we can no longer be, what was before promoted(that I know of) since 2020-2024:
- layer 0 project
- secure bridging solution
- application chain

I would recommend we choose a path within community and build a project that we can market. Changing narrative so often by chasing what is popular will make us look fishy, lost and not worth investing into. 
Now would be the time to market what we built in bear market. What is on with MSGPORT? I don't see any application joining if we have nothing to offer.

I would propose to gather all team to work on single project that will do one thing and one thing only bring value to token holders of RING. 
 I would only keep two DAOs. We have too much DAOs. Ring DAO would govern RING part and KTON Dao KTON part. Other DAOs like DCDAO I would terminate and put it under RING DAO as would be better to involve whole community in decision making and biweekly meetings, not just DCDAO. More people know more...Community is at loss imho, by not participating in calls with you guys builders. We can all learn the most from you. This would also build much needed trust for newcomers as now most trust comes from first front line the moderators. 

CRAB DAO and CKTON DAO hugely depend on how KUSAMA will move forward if at all.... 

These are my 5 cents. Darwinia/Ring DAO is becoming very confusing for us ambassadors let alone for community, That is why this debate much appreciated and a proper solution should be found.

**Reply by sonoferin2000** [reply]
I agree that the situation with the daos is very confusing, and an equitable solution should be found to deal with it to simplify things. its logical that ring/crab should be united and kton/ckton be united under their respective umbrellas.

**Comment by sonoferin2000** [comment]
@hackfisher Thanks a lot for bringing up this discussion. it is a lot to process. Will give more feedback to this over the weekend.

**Comment by Divblaze** [comment]
One thing I recommend we create a utility video for these products to explain how to use to make things easier for new persons that will love to integrate on the ecosystem
